### DISCUSSION TITLE
Images refusing to show in Github

### DISCUSSION THREAD

**Discussion by jntak** [community/community#143123] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I wonder if any one else is experiencing this but for a while now images including my profile picture are no loading and showing in GitHub. Both on mobile and web apps. How can I solve this problem?

**Comment by thomassiard** [comment]
Clear Cache and Cookies: Sometimes, clearing your browser's cache and cookies can fix loading issues. Go to your browser settings and clear the cache, then try reloading GitHub.

Check Your Internet Connection: Ensure that your internet connection is stable. Try accessing other websites to see if they load properly.

Try a Different Browser: If you're using a specific browser, try accessing GitHub using a different one to see if the issue persists.

Disable Browser Extensions: Some extensions might interfere with how pages load. Temporarily disable them and check if that solves the problem.

Update Your Apps: If you're using the GitHub mobile app, make sure it's updated to the latest version. Sometimes, bugs in older versions can cause issues like this.

Check GitHub Status: Occasionally, GitHub may experience outages or issues that affect loading. You can check their [status page](https://www.githubstatus.com/) for any ongoing incidents.

Log Out and Log Back In: Sometimes, simply logging out of your account and logging back in can refresh your session and fix the problem.
